### ISSUE TITLE
Перестал работать поиск по наименованию изделия

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1926]
Originally reported on Google Code with ID 1926

```
Начиная с версий экзешника более 5736, перестал работать поиск по
наименованию изделия при выборе остатков.
```

Reported by `gs1994` on 2010-03-21 17:24:38

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-03-21 17:26:33
- **Labels added**: Tree
- **Labels removed**: GomelTree

**Comment by gsbelarus** [comment]
Reported by `flakekun` on 2010-03-22 16:13:58
- **Status changed**: `Duplicate`
- **Merged into**: #1789

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2011-02-23 10:30:11
- **Labels added**: GomelTree
- **Labels removed**: Tree
